src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import Todo from '../../types/Todo';
+
+import users from '../../api/users';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const currentUser = users.find(user => user.id === todo.userId);
+
+  return (
```

Comment:
You should have prepared information about user in root component and sent user as a parameter or as a part of `todo`.
[checklist (5)](https://github.com/mate-academy/react_add-todo-form/blob/master/newChecklist.md)